### PR TITLE
Move all bundling into vscode-lit-plugin

### PR DESCRIPTION
**Pull request by rictic**
This way downstream users of ts-lit-plugin are free to bundle it with their preferred settings.

### CODE REVIEW

File: packages/vscode-lit-plugin/package.json [hunk 1]
```diff
@@ -122,13 +122,16 @@
 			"dependencies": [
 				"../lit-analyzer:build"
 			],
-			"command": "node esbuild.script.js",
+			"command": "node esbuild.script.mjs",
 			"files": [
 				"src/**/*",
-				"esbuild.script.js"
+				"../ts-lit-plugin/src/**/*",
+				"../ts-lit-plugin/index.js",
```
Comment (rictic) [inline packages/vscode-lit-plugin/package.json:129]: @aomarks is this kosher, or should I put this script at the toplevel of the monorepo?

Comment (aomarks) [reply packages/vscode-lit-plugin/package.json:129]: Should be fine, input files can reach out of the package. It's only output files that currently can't.

File: packages/vscode-lit-plugin/esbuild.script.mjs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  target: "es2017",
+  format: "cjs",
+  color: true,
+  mainFields: ["module", "main"]
+});
+
+try {
+  await Promise.all([buildUs, buildTsLitPlugin]);
+} catch (e) {
+  process.exit(1);
```
Comment (aomarks) [inline packages/vscode-lit-plugin/esbuild.script.mjs:32]: Log the error?

**Review by aomarks — APPROVED** [review]

File: packages/vscode-lit-plugin/esbuild.script.mjs (hunk 2 above)
Comment (rictic) [reply packages/vscode-lit-plugin/esbuild.script.mjs:32]: This is the recommended style from the esbuild docs, the error has some useful info but it's mostly a retread of the error that's already been logged at that point.

On the other hand... simplicity is nice, and this is hella fast already. Switched to just awaiting directly.

**Review by aomarks — APPROVED** [review]

**Review by justinfagnani — APPROVED** [review]

**Pull request merged**